src/pages/UserPage/UserPage.jsx

```diff
@@ -43,16 +52,16 @@ const UserPage = () => {
     const fetchUser = async () => {
       dispatch(fetchUserRequest());
       try {
-        const response = await axios.get(`${apiUrl}/users/${id}`);
+        const response = await axios.get(`${apiUrl}/users/profile`, config);
         dispatch(fetchUserSuccess(response.data.data));
-        const { 
+        const {
           name, 
-          last_name, 
-          email, 
-          password, 
-          phone_numbers, 
-          addresses, 
-          city 
+          last_name,
+          email,
+          password,
+          phone_numbers,
+          addresses, //try later addresses: address ,
```

Comment:
is this comment necessary?

Reply:
No, it was a comment for myself that i forgot to try it and erase it. 

Resolved